boards: STM32L5: Update TFM related instructions (#33499)
zephyrproject-rtos/zephyr: boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

Building a secure/non-secure with Arm |reg| TrustZone |reg|
-----------------------------------------------------------

The TF-M integration sample :ref:`tfm_ipc` can be run on a ST Nucleo L552ZE Q.

> In TFM configuration, Zephyr is run on the non-secure side. A non-secure image

Reviewer: use consistently, TF-M, instead of TFM

Reviewer: non-secure domain

Author: Fixed, txs for the review

> could be generated using ``nucleo_l552ze_q_ns`` as build target.

Reviewer 2: ```suggestion
can be generated using ``nucleo_l552ze_q_ns`` as build target.
```

.. code-block:: bash

>    $ west build -b nucleo_l552ze_q_ns

Reviewer 3: I assume the sample project path `samples/tfm_integration/tfm_ipc` is required in the build command?

Author: Sure, I'm adding path/to/source/directory (as this is board documentation, let's make it generic)

Reviewer 3: Looks good to me.

Note: When building the ``*_ns`` image with TF-M, ``build/tfm/postbuild.sh`` bash script
is run automatically in a post-build step to make some required flash layout changes.

> Once the build is completed, run the following script to perform option bytes initialization.

Reviewer 2: I'm not sure of the exact sense here, but if we're changing 'option bytes' this might be clearer:

```suggestion
Once the build is completed, run the following script to initialize the option bytes.
```

.. code-block:: bash

   $ build/tfm/regression.sh

Finally, to flash the board, run:

.. code-block:: bash

   $ west flash --hex-file build/tfm_merged.hex

Note: Check the ``build/tfm`` directory to ensure that the commands required by these scripts

> (``readlink``, etc.) are available on your system. Please also check ``STM32_Programmer_CLI``
> used for initialization is available in the PATH.

Reviewer 2: ```suggestion
(``readlink``, etc.) are available on your system. Please also check ``STM32_Programmer_CLI``
(which is used for initialization) is available in the PATH.
```
